### ISSUE TITLE
Works dropdown in navbar should open on mouse over.

### ISSUE THREAD

**Issue by g1st** [chingu-voyages/v8-bears-team-14#27]
I think this would be better ux instead of a click? Also hide menu when cursor leaves dropdown menu area. Not sure if this possible in material-ui though.

**Comment by nikrb** [comment]
PWA's are mobile first offline, ideally. afaik there is no hover on touch devices.

**Comment by g1st** [comment]
Oops, are we going for PWA's functionality? Is it possible to have dropdown by hover on big screen and by touch on mobile?

**Comment by nikrb** [comment]
well most apps seem to go mobile first? I'm sure it's possible to change behaviour, but maybe later? :)

**Comment by g1st** [comment]
Yes, I would love to have this behaviour! Not sure how much time do we have and if it is difficult to implement but would love it for sure! Hopefully later!
